src/scripts.d/66_device_Network

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+set -e
+
+EXIT_OK=0
+
+# All the network interfaces
+INTERFACES=$(ls /sys/class/net)
+
+get_lan_info() {
+    for IFACE in $INTERFACES; do
+        # Exclude loopback
+        if [ $IFACE = "lo" ]; then
+            continue
+        fi
+
+        MAC=$(cat /sys/class/net/"$IFACE"/address)
+        ENABLED=$(cat /sys/class/net/"$IFACE"/operstate | grep -q 'up' && echo 1 || echo 0)
+
+        IP4_LIST=$(ip -4 addr show dev "$IFACE" 2>/dev/null | awk '
+            $1 == "inet" {
+                ip_list = (ip_list ? ip_list "," : "") "\"" $2 "\"";
+            }
+            END {
+                print ip_list
+            }
+        ')
+
+        IP6_LIST=$(ip -6 addr show dev "$IFACE" 2>/dev/null | awk '
+            $1 == "inet6" && $2 !~ /^fe80/ {
+                ip_list = (ip_list ? ip_list "," : "") "\"" $2 "\"";
+            }
+            END {
+                print ip_list
+            }
+        ')
+
+        printf 'LAN_%s={"name":"%s","ip4":[%s],"ip6":[%s],"mac":"%s","enabled":%s}\n' "$IFACE" "$IFACE" "${IP4_LIST:-}" "${IP6_LIST:-}" "${MAC:-}" "${ENABLED:-0}"
+
+    done
+}
+
+get_wan_info() {
+    TARGET="8.8.8.8"  # PING target
+    COUNT=1           # PING count
+    DEADLINE=5        # PING deadline
+
+    if ! command -v ping >/dev/null 2>&1; then
+        PING_STATUS="Ping not found"
+    else
+        ping -c "$COUNT" -w "$DEADLINE" "$TARGET" > /dev/null 2>&1
+        EXIT_CODE=$?
+        if [ $EXIT_CODE -eq 0 ]; then
+            PING_STATUS="OK"
+        else
+            PING_STATUS="Failed"
+        fi
+    fi
+
+    if command -v curl >/dev/null 2>&1; then
+        GLOBAL_IP=$(curl -s --connect-timeout 5 ifconfig.me)
+    fi
+    printf 'WAN={"globalIp":"%s","pingStatus":"%s"}\n' "${GLOBAL_IP:-}" "${PING_STATUS}"
```

Comment:
Where did the "globalIp" name come from? I'm wondering if "publicIp" would be a more common term.